hassio/src/addon-view/info/hassio-addon-info.ts

```diff
@@ -170,11 +173,25 @@ class HassioAddonInfo extends LitElement {
                   iconClass="update"
                 ></hassio-card-content>
                 ${!this.addon.available
-                  ? html`
-                      <p>
-                        This update is no longer compatible with your system.
-                      </p>
-                    `
+                  ? !this.addon.arch.find((arch) =>
+                      this.supervisor.info.supported_arch.includes(arch)
+                    )
+                    ? html`
+                        <p>
+                          The architecture on your system
+                          (${this.supervisor.supervisor.arch}) is no longer
```

Comment:
We don't know if it is "no longer", or "never has been".

```suggestion
                          The architecture on your system
                          (${this.supervisor.supervisor.arch}) is not
```

Reply:
So not display archs at all?